src/transformers/pipelines/audio_utils.py

```diff
@@ -149,10 +164,10 @@ def ffmpeg_microphone_live(
         format_for_conversion (`str`, *optional*, defaults to `f32le`):
             The name of the format of the audio samples to be returned by ffmpeg. The standard is `f32le`, `s16le`
             could also be used.
-        ffmpeg_input_device (`str`, *optional*):
-            The identifier of the input device to be used by ffmpeg (i.e. ffmpeg's '-i' argument). If unset,
-            the default input device will be used. See `https://www.ffmpeg.org/ffmpeg-devices.html#Input-Devices`
-            for how to specify and list input devices.
+        ffmpeg_additional_args (`list`, *optional*):
```

Comment:
Same comments here - there should be an example of how to pass in multiple arguments in the docstring 

```suggestion
        ffmpeg_additional_args (`List[str]`, *optional*):
```